app/styles/templates/project/index.scss

```diff
@@ -1,14 +1,25 @@
 .project-wrapper {
+  display: flex;
+  flex-direction: column;
   min-height: 50vh;
   width: 100%;
+
+  &--index {
```

Comment:
huh I never knew this was a rule you could use (I see `project-wrapper--full`).  Cool!